src/components/Company/Offers/Manage/CompanyOffersManagementWidget.spec.js

```diff
@@ -32,7 +37,15 @@ describe("App", () => {
         },
     ];
 
+    beforeEach(() => {
+        useSession.mockReturnValue({ data: { company: { name: "company1", _id: "company_id" } }, isLoggedIn: true });
+    });
+
     test("Renders Loading", () => {
+        companyOffersService.fetchCompanyOffers.mockImplementationOnce(() => new Promise((resolve) => setTimeout(() => {
+            resolve();
+        }, 1000)));
```

Comment:
This test is dependant on the fact that the render takes less than 1000ms, that may not happen due to concurrent test execution making it slower, or some other reason. Instead, don't call the resolve function (remove the setTimeout as well. This way you return a Promise that will never resolve, and it will be in loading state forever

Reply:
Yeah I was thinking about refactoring this test as well :smile:.. Done